File: gdscript/src/main/java/gdscript/psi/GdSetDecl.java
package gdscript.psi;

import com.intellij.psi.PsiElement;
import org.jetbrains.annotations.Nullable;

public interface GdSetDecl extends PsiElement {

  @Nullable
  GdEndStmt getEndStmt();

  @Nullable
  GdSetMethodIdRef getSetMethodIdNm();

  @Nullable
  GdStmtOrSuite getStmtOrSuite();

  @Nullable
  GdTyped getTyped();

  @Nullable
  GdVarNmi getVarNmi();

}

File: gdscript/src/main/kotlin/gdscript/psi/GdSetDecl.kt
package gdscript.psi

import com.intellij.psi.PsiElement

interface GdSetDecl : PsiElement {
    val endStmt: GdEndStmt?

    val setMethodIdNm: GdSetMethodIdRef?

    val stmtOrSuite: GdStmtOrSuite?

    val typed: GdTyped?

    val varNmi: GdVarNmi?
}
